util/runtests.sh

```diff
@@ -146,6 +163,27 @@ for f in $testfiles; do
 
         status=$?
         echo $f returned $status >> "$TESTLOG" 2>&1
+        if [ "$pytest" = True ]; then
+          if [ "$status" -eq 124 ]; then
+              echo "Test $f timed out after $MAXTIME seconds" >> "$TESTLOG"
+              echo "Test $f timed out after $MAXTIME seconds" >> "$TIMEOUTSUMMARY"
```

Comment:
Maybe it'd be easier to parse for later scripts if the output looks more like the pytest short summary.
Eg: 
```
FAILED src/odemis/driver/test/piezomotor_test.py::TestPMD401OpenLoop::test_queued - OSError: F...
```
So you could output something like:
```
TIMEOUT $f
```
Then, eventually it has to be picked up by the "email writer". I don't know precisely where is this code...

Reply:
I think it'd still be easier parse if the output is something like "TIMEOUT $f" or "FAILED $f" , but I leave it up to you. You can make a separate PR for updating the filter to read this part of the summary.